e2e/test/scenarios/collections/instance-analytics.cy.spec.js

```diff
@@ -0,0 +1,133 @@
+      cy.visit("/");
+      navigationSidebar().findByText(ANALYTICS_COLLECTION_NAME).click();
+      cy.findByTestId("pinned-items")
+        .findByText("Users")
+        .scrollIntoView()
+        .click();
+
+      cy.findByTestId("TableInteractive-root").within(() => {
+        cy.findByTextEnsureVisible("admin@metabase.test");
+        cy.findByTextEnsureVisible("Robert Tableton");
+        cy.findByTextEnsureVisible("Read Only Tableton");
+      });
+    });
+
+    it("should default to saving saving audit content in custom reports collection", () => {
+      cy.log("saving edited question");
+
+      navigationSidebar().findByText(ANALYTICS_COLLECTION_NAME).click();
+
+      cy.findByTestId("pinned-items")
+        .findByText("Users")
+        .scrollIntoView()
+        .click();
+
+      popover().findByText("Filter by this column").click();
+      cy.wait("@fieldValues");
+      popover().findByText("Tableton").click();
+      popover().button("Add filter").click();
+
+      cy.wait("@datasetQuery");
+      cy.findByTestId("question-row-count").findByText("Showing 6 rows");
+
+      cy.wait("@datasetQuery");
+      cy.findByTestId("question-row-count").findByText("Showing 6 rows");
+
+      cy.findByTestId("qb-header").findByText("Save").click();
+
+      modal().within(() => {
+        cy.findByTextEnsureVisible("Custom reports");
+        cy.button("Save").click();
+      });
+
+      cy.wait("@saveCard").then(({ response }) => {
+        expect(response.statusCode).to.eq(200);
+      });
+
+      modal().button("Not now").click();
+
+      cy.log("saving copied question");
+
+      navigationSidebar().findByText(ANALYTICS_COLLECTION_NAME).click();
+
+      cy.findByTestId("pinned-items")
+        .findByText("Users")
+        .scrollIntoView()
+        .click();
+
+      cy.findByTestId("qb-header").icon("ellipsis").click();
+
+      popover().findByText("Duplicate").click();
+
+      modal().within(() => {
+        cy.findByTextEnsureVisible("Custom reports");
+        cy.button("Duplicate").click();
+      });
+
+      cy.wait("@saveCard").then(({ response }) => {
+        expect(response.statusCode).to.eq(200);
+      });
+
+      modal()
+        .button(/Duplicate/i)
+        .should("not.exist");
+      modal().button("Not now").click();
+
+      cy.log("saving copied dashboard");
+
+      navigationSidebar().findByText(ANALYTICS_COLLECTION_NAME).click();
+
+      cy.findByTestId("pinned-items").findByText("Single user view").click();
+
+      cy.findByTestId("dashboard-header").findByText("Make a copy").click();
+
+      modal().within(() => {
+        cy.findByTextEnsureVisible("Custom reports");
+        cy.button("Duplicate").click();
+      });
+
+      cy.wait("@copyDashboard").then(({ response }) => {
+        expect(response.statusCode).to.eq(200);
+      });
+    });
+
+    it.skip("should not allow moving or archiving custom reports collection", () => {
```

Comment:
this is a test for https://github.com/metabase/metabase/pull/35230, will unskip it when that merges